lib/export/jdl_exporter.js

```diff
@@ -18,11 +18,155 @@
  */
 
 const fs = require('fs');
+const path = require('path');
+const applicationOptions = require('../core/jhipster/application_options');
+const unaryOptions = require('../core/jhipster/unary_options');
+const JSONFileReader = require('../reader/json_file_reader');
+const JSONReader = require('../reader/json_reader');
+const JSONToJdlApplicationConverter = require('../converters/json_to_jdl_application_converter');
+const FileUtils = require('../utils/file_utils');
+const JDLObject = require('../core/jdl_object');
 
 module.exports = {
-  exportToJDL
+  exportToJDL,
+  loadJhipsterConfigAndEntities
```

Comment:
I wouldn't have put it here (it's a matter of responsibility), but rather in an other file.
Note: there the `JSONFileReader::readEntityJSON` function... they should be a refactor here.

Reply:
I am not sure to fully understand what you mean.

Reply:
I don't see why `loadJhipsterConfigAndEntities` should be in the JDLExporter.

Reply:
Then should we move all the others functions as well, since they are just used to load the configurations ?

Reply:
Yep. For now, let's keep it there, there'll be a small refacto later anyway.